blog.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="zxx" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <title>Adopt A Dog | Blog</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <header>
+    <div>
+      <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 505.766 153.879">
+        <defs>
+          <style>
+            .cls-1 {
+              fill: none;
+            }
+
+            .cls-2 {
+              clip-path: url(#clip-path);
+            }
+
+            .cls-3 {
+              font-size: 62.641px;
+              font-family: WorkSans-Black, Work Sans;
+              letter-spacing: -0.009em;
+            }
+
+            .cls-3,
+            .cls-7 {
+              fill: #fff;
+            }
+
+            .cls-4 {
+              letter-spacing: -0.03em;
+            }
+
+            .cls-5 {
+              letter-spacing: -0.098em;
+            }
+
+            .cls-6 {
+              letter-spacing: 0em;
+            }
+          </style>
+          <clipPath id="clip-path" transform="translate(-1 -1.168)">
+            <rect class="cls-1" x="1" y="1.168" width="502.59" height="135.832" />
+          </clipPath>
+        </defs>
+        <title>adoptadog-logo</title>
+        <g class="cls-2"><text class="cls-3" transform="matrix(0.966, 0, -0.259, 0.966, 117.172, 133.124)">ADO<tspan class="cls-4" x="131.984" y="0">P</tspan>
```

Comment:
This SVG should be imported in as an <img >tag. You can use the src to link to the file.